.github/workflows/ci_tests_dev.yaml

```
@@ -28,7 +30,7 @@ jobs:
       fail-fast: false
       matrix:
         python-version: [3.9]
-        os: [ubuntu-20.04, macOS-10.15, windows-latest]
+        os: [ubuntu-latest, macOS-latest, windows-latest]
```

Do you want to test out macOS Big Sur 11.0? I see it listed at https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#github-hosted-runners, but not sure what the status is on https://github.com/GenericMappingTools/gmt/issues/4442. We could give it a try though and see if it works :smile:

```suggestion
        os: [ubuntu-latest, macOS-11.0, windows-latest]
```

---

Actually, I already used 'macOS-11.0' in the gmt-release-testbot. macOS bundle, homebrew and macports all work well. See https://github.com/GenericMappingTools/gmt-release-testbot/actions/runs/544167288 (you can check the macos version in the "Set up job" step).

---

Ah nice, so maybe it does work then, let's give it a go.

---

As an aside, do we want to add an osx-arm64 build for the conda-forge gmt-feedstock, following https://conda-forge.org/blog/posts/2020-10-29-macos-arm64/#how-to-add-a-osx-arm64-build-to-a-feedstock?